src/App.tsx

```diff
@@ -1,61 +1,156 @@
+import { useEffect, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import { TodoList } from './components/TodoList/TodoList';
+
+function getUser(userId: number): User | undefined {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || undefined;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [selectedTitle, setTitle] = useState('');
+  const [selectedUser, setUser] = useState('');
```

Comment:
so here we can simplify our names
```suggestion
  const [title, setTitle] = useState('');
  const [user, setUser] = useState('');
```